용재/17413.py

```diff
@@ -0,0 +1,27 @@
+getText = input()
+
+result = []
+stack = []
+
+flag = False
+
+for each in getText:
+  if each == '>':
+    result.append(each)
+    flag = False
+  elif each == '<' or flag == True:
```

Comment:
파이썬에서 조건문을 사용할 때 boolean 타입의 경우에는 `if flag == True` 이러한 형식 대신
`if flag`와 같이 사용 하는 것을 권장하고 있다고 알고 있습니다!

```suggestion
  elif each == '<' or flag:
```

Reply:
아하 확실히 없어도 간결하게 의도를 전달 할 수 있겠네요 감사합니다!